driver-core/src/main/java/com/datastax/driver/core/policies/DCAwareRoundRobinPolicy.java

```diff
@@ -395,24 +354,10 @@ public void onUp(Host host) {
                 return;
         }
         dcHosts.addIfAbsent(host);
-
-        CopyOnWriteArrayList<Host> dcSuspected = perDcSuspectedHosts.get(dc(host));
-        if (dcSuspected != null)
-            dcSuspected.remove(host);
     }
 
     @Override
     public void onSuspected(Host host) {
```

Comment:
With the suspect state gone should we update [JAVA-616](https://datastax-oss.atlassian.net/browse/JAVA-616) to remove this from StateListener or will we plan to keep it for some later use?